deploy/katello.spec

```diff
@@ -108,7 +100,34 @@ chmod +x %{buildroot}%{homedir}/script/*
 # install man page
 install -m 644 man/katello-service.8 %{buildroot}/%{_mandir}/man8
 
-%post
+%clean
+rm -rf %{buildroot}
```

Comment:
ideally we'll want to use the macro forms of common shell commands (in this case: `%{__rm}`) for ultimate portability, but that's probably low priority.